magma_access_gateway_installer/__init__.py

```diff
@@ -31,44 +32,53 @@ def main():
     network_interfaces.remove("lo")
 
     args = cli_arguments_parser(sys.argv[1:])
-    validate_args(args, network_interfaces)
-
-    preinstall_checks = AGWInstallerPreinstallChecks(network_interfaces)
-    network_configurator = AGWInstallerNetworkConfigurator(
-        network_interfaces, args.dns, args.ip_address, args.gw_ip_address, args.sgi, args.s1
-    )
-    service_user_creator = AGWInstallerServiceUserCreator()
-    installation_service_creator = AGWInstallerInstallationServiceCreator()
-
-    preinstall_checks.preinstall_checks()
-    preinstall_checks.install_required_system_packages()
-
-    if not args.skip_networking:
-        network_configurator.update_interfaces_names()
-        network_configurator.configure_dns()
-        network_configurator.create_interfaces_config_files()
-    network_configurator.remove_netplan()
-    network_configurator.enable_networking_service()
-
-    service_user_creator.create_magma_user()
-    service_user_creator.add_magma_user_to_sudo_group()
-    service_user_creator.add_magma_user_to_sudoers_file()
-
-    if network_configurator.reboot_needed:
-        installation_service_creator.create_magma_agw_installation_service()
-        installation_service_creator.create_magma_agw_installation_service_link()
-        logger.info(
-            "Rebooting system to apply pre-installation changes...\n"
-            "Magma AGW installation process will be resumed automatically once machine is back up."
+    try:
+        validate_args(args, network_interfaces)
+    except ArgumentError as e:
+        print(e.message)
```

Comment:
Why are we using `print` instead of logger?

Reply:
Because logger is configured to log to the journal.

Reply:
It's a good point though, we might want to use logging for both writing to the journal and printing, I'll check into this

Reply:
Now fixed